query/grouped_result.go

```diff
@@ -6,6 +6,7 @@ import (
 )
 
 // GroupedResults is a collection of result groups.
+// TODO(wjang): add JSON marshaling / unmarshaling
```

Comment:
Comments start w/ upper case and should end w/ a period as they are complete sentences.